### ISSUE TITLE
retrain the classifier model

### ISSUE THREAD

**Issue by drjova** [cern-sis/issues-inspire#432]
retrain classifier following the guide https://inspire.docs.cern.ch/operations/classifier/ and using the new PC

### Implementation
- [x] Implement previous split for Training, Test and Validation Set
    - [x] Use `create_dataset.py` to create Training and Test split
    - [x] Use Dataloaders in classifier to split for training and validation set
    - [x] Adjust path structure to new splits
- [x] Implement ULMfit architecture
    - [x] Language Model implementation 
        - [x] Create Vocabulary using min and max vocab size
        - [x] Load pre-trained Language Model
        - [x] Fine-tune pre-trained LM with Inspire Data
        - [x] Save fine-tuned LM encoder layer
    - [x] Classifier implementation
        - [x] Load classifier model
        - [x] Replace encoder layer
        - [x] Load and split Inspire data into validation and training set
        - [x] Train using Inspire Data
        - [x] Implement `predict` method
- [x] Cleanup unnecessary code from old implementation
- [x] Guarantee that the cli is useable as before (get all tests running)
- [ ] Fine-tune parameters for best results (also try one to one copy of old parameters)
- [x] Write readme with step by step instructions on how to use the classifier

### Training
- [x] Pull training data for last x Years 
- [x] Train classifier model with the new data

**Comment by PascalEgn** [comment]
Monitor Memory usage of classifier on QA using Grafana. 
If there are still problems, either reduce number of Gunicorn workers or use https://docs.gunicorn.org/en/stable/settings.html#max-requests to automatically restart them (to dodge possible memory leaks)
